test/edu-benefits/1990e/config/educationHistory.unit.spec.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-addons-test-utils';
+
+import { DefinitionTester, submitForm } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/edu-benefits/1990e/config/form';
+
+describe('Edu 1990e educationHistory', () => {
+  const { schema, uiSchema } = formConfig.chapters.educationHistory.pages.educationHistory;
+  const definitions = formConfig.defaultDefinitions;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          data={{}}
+          uiSchema={uiSchema}
+          definitions={definitions}/>
+    );
+    const fields = ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'input');
+    expect(fields.length).to.equal(8);
+  });
+  it('should have no required inputs', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          data={{}}
+          uiSchema={uiSchema}
+          definitions={definitions}/>
+    );
+    const formDOM = findDOMNode(form);
+    submitForm(form);
+    expect(Array.from(formDOM.querySelectorAll('.usa-input-error'))).to.be.empty;
+    // TODO: find out why onSubmit is not called
+    // expect(onSubmit.called).to.be.true;
```

Comment:
This returns false for some reason--am trying to figure out why. 

Also, will go back and see if Alex updated any of the issues with which fields are required.